api/config.py

```diff
@@ -151,6 +164,18 @@ class Configuration(CoreConfiguration):
             "key": HELP_URI,
             "label": _("Patron support custom integration URI")
         },
+        {
+            "key": LARGE_COLLECTION_LANGUAGES,
+            "label": _("The primary languages represented in this library's collection")
+        },
+        {
+            "key": SMALL_COLLECTION_LANGUAGES,
+            "label": _("Other major languages represented in this library's collection")
+        },        
+        {
+            "key": TINY_COLLECTION_LANGUAGES,
+            "label": _("Other languages in this library's collection")
+        },        
```

Comment:
Sorry, that's not quite right. Type "list" can have fixed options, or not, and if it doesn't have them you get to add your own. Type "select" only lets you pick one thing from fixed options.